src/btree.c

```diff
@@ -3145,100 +3145,104 @@ btree_count_in_range_by_iterator(cache        *cc,
  *-----------------------------------------------------------------------------
  */
 void
-btree_print_locked_node(btree_config          *cfg,
-                        uint64                 addr,
-                        btree_hdr             *hdr,
-                        platform_stream_handle stream)
+btree_print_locked_node(btree_config        *cfg,
+                        uint64               addr,
+                        btree_hdr           *hdr,
+                        platform_log_handle *log_handle)
```

Comment:
Minor request / suggestion: As long as you are overhauling these interfaces, can you consider making the output stream / log-handle arg; i.e. `platform_log_handle *log_handle` the 1st arg to the print method?

That will keep the interface somewhat consistent with the underlying `platform_log*()` APIs, too, which take the output stream as the 1st arg.

It will be good to consistently move to that convention as we develop / retrofit print methods.